docs/how-to-guides/block-tutorial/writing-your-first-block-type.md

```diff
@@ -1,137 +1,174 @@
-		$asset_file['version']
-	);
+The functions of a static block is defined in JavaScript, however the settings and other metadata should be defined in a block.json file.
 
-	register_block_type( 'gutenberg-examples/example-01-basic-esnext', array(
-		'api_version' => 2,
-		'editor_script' => 'gutenberg-examples-01-esnext',
-	) );
+Here are the basic settings:
+
+-   `title`: Block title shown in inserter
+-   `name`: Unique name defines your block
+-   `category`: Category in inserter (text, media, design, widgets, theme, embed)
+-   `icon`: Dashicon icon displayed for block
+-   `editorScript`: JavaScript file to load for block
+
+The `block.json` file should be added to your plugin. To start a new plugin, create a directory in `/wp-content/plugins/` in your WordPress.
+
+Create a basic `block.json` file there:
+
+{% codetabs %}
+{% JSX %}
 
+```json
+{
+	"apiVersion": 2,
+	"name": "gutenberg-examples/example-01-basic-esnext",
+	"title": "Example: Basic (ESNext)",
+	"icon": "universal-access-alt",
+	"category": "layout",
+	"editorScript": "file:./build/index.js"
 }
-add_action( 'init', 'gutenberg_examples_01_register_block' );
 ```
 
-Note the above example, shows using the [wp-scripts build step](/docs/how-to-guides/javascript/js-build-setup/) that automatically sets dependencies and versions the file.
+{% Plain %}
+
+```json
+{
+	"apiVersion": 2,
+	"title": "Example: Basic",
+	"name": "gutenberg-examples/example-01-basic",
+	"category": "layout",
+	"icon": "universal-access-alt",
+	"editorScript": "file:./block.js"
+}
+```
 
-If you were using the plain code, you would specify `array( 'wp-blocks', 'wp-element' )` as the dependency array. See the [example 01](https://github.com/WordPress/gutenberg-examples/blob/HEAD/01-basic/index.php) in Gutenberg Examples repository for full syntax.
+{% end %}
 
--   **`wp-blocks`** includes block type registration and related functions
--   **`wp-element`** includes the [WordPress Element abstraction](/packages/element/README.md) for describing the structure of your blocks
+### Step 2: Register block in plugin
 
-## Registering the Block
+With the `block.json` in place, the registration for the block is a single function call in PHP, this will setup the block and JavaScript file specified in the `editorScript` property to load in the editor.
 
-With the script enqueued, let's look at the implementation of the block itself:
+Create a full plugin file, `index.php` like the following, the same PHP code works for both JSX and Plain code.
+
+```php
+<?php
+/**
+ * Plugin Name: Gutenberg examples 01
+ */
+function gutenberg_examples_01_register_block() {
+	register_block_type( __DIR__ );
+}
+add_action( 'init', 'gutenberg_examples_01_register_block' );
+```
+
+### Step 3: Block edit and save functions
+
+The `editorScript` that loads in the editor calls a register function with a matching `name` specified in `block.json` and defines two important functions for the block, the `edit` and `save` functions.
```

Comment:
This is a little unclear to me. I wonder if we only need to explain at a high level?
```suggestion
The `editorScript` entry is enqueued automatically in the block editor. This file contains the JavaScript portion of the block registration and defines two important functions for the block, the `edit` and `save` functions.
```